### PR TITLE
fix(RELEASE-1025): stop cloning the repo, install library from pypi

### PR DESCRIPTION
**Pull request by pkhander**
pubtools-content-gateway was upgraded - https://github.com/release-engineering/pubtools-content-gateway/commit/82d7e652791e000616d6c8a6071d721a3019c588

This commit aims to install library directly from [pypi](https://pypi.org/project/pubtools-content-gateway/) instead of cloning the repo.

JIRA: RELEASE-1025

### CODE REVIEW

**Review by johnbieren — DISMISSED** [review]

File: Dockerfile
Comment (mmalina) [inline Dockerfile:1]: Maybe pubtools-content-gateway can also be added to requirements.txt now?

Comment (pkhander) [reply Dockerfile:1]: added!

Comment (mmalina) [reply Dockerfile:1]: Thanks!

**Review by mmalina — APPROVED** [review]

**Pull request merged**